src/components/alert/alert.tsx

```diff
@@ -126,8 +126,17 @@ export class Alert {
     this.requestedIcon = setRequestedIcon(StatusIcons, this.icon, this.color);
   }
 
+  componentDidLoad(): void {
+    this.transitionRunEvent();
+  }
```

Comment:
This will only wire up the event listener once in the page lifecycle so if the element is disconnected from the dom and relocated, the listener will be destroyed.  You probably want this to be inside `connectedCallback` instead.